R/parallelization.R

```diff
@@ -13,97 +68,99 @@
 #' according to your individual setup and task, but a rough guide would be in
 #' the order of 100 microseconds to 1 millisecond for each map iteration.
 #'
-#' # Daemons settings
+#' @section Daemons Settings:
```

Comment:
Out of interest, why did you change these?

Reply:
Are you referring to the case of the section title? I wasn't aware that we were team sentence case 😄

Reply:
Here I meant from `#` to `@section`

Reply:
Just to make it consistent with other the other usage in the same function. I probably copied the # over originally from the progress bars docs.